ui/packages/docs/src/components/Fields/character_data.json

```diff
@@ -172,7 +172,15 @@
       "fields": [
         ".travelerhydro.droplets"
       ],
-      "desc": "Number of Sourcewater droplets created by Traveler."
+      "desc": "Number of Sourcewater droplets on the field."
```

Comment:
why?

Reply:
I don't quite understand the point of differentiating between those created by HydroMC vs anyone else. Besides the unshared 12 gadget limit, they behave identically

Reply:
i initially created it for testing a1, but for some reason i decided to keep it.
if you need the number of sourcewater droplets, create a field `.gadgets.sourcewater.count` (see dendrocore implementation)

Reply:
Should I remove this param from the HMC then? And still keep the one from Neuvillette, since that one uses Neuvillettes position to check which ones are in range?